composer.json

```diff
@@ -76,7 +77,11 @@
         "monolog/monolog": "~1.11",
         "ircmaxell/password-compat": "~1.0",
         "ocramius/proxy-manager": "~0.4|~1.0",
-        "egulias/email-validator": "~1.2"
+        "egulias/email-validator": "~1.2",
+        "phpdocumentor/reflection": "~1.0"
```

Comment:
You've added a `conflict` block below for versions < 1.0.7. Shouldn't we change the version constraint instead to `^1.0.7` and drop the `conflict` block?